<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>616</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>50000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="uwidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #dddddd</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="dwidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="bHead">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Head</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bResize">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Resize</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bRembg">
           <property name="text">
            <string>RemBG</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bCopy">
           <property name="text">
            <string>Copy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bHFlip">
           <property name="text">
            <string>HFlip</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bGPEN">
           <property name="text">
            <string>GPEN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bInterpolate">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Interpolate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="info">
           <property name="styleSheet">
            <string notr="true">color: blue</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionReload_Files"/>
   </widget>
   <widget class="QMenu" name="menu_Tool">
    <property name="title">
     <string>&amp;Tool</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Delete"/>
    <addaction name="action_HFlip"/>
    <addaction name="action_GPen"/>
    <addaction name="action_Resize"/>
    <addaction name="action_Rotate"/>
    <addaction name="actionRe_name_Files"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Tool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Folder">
   <property name="text">
    <string>Open Folder</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_HFlip">
   <property name="text">
    <string>&amp;HFlip</string>
   </property>
  </action>
  <action name="action_GPen">
   <property name="text">
    <string>&amp;GPen</string>
   </property>
  </action>
  <action name="action_Rotate">
   <property name="text">
    <string>ro&amp;Tate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Resize">
   <property name="text">
    <string>&amp;Resize</string>
   </property>
  </action>
  <action name="actionRe_name_Files">
   <property name="text">
    <string>Re&amp;name Files</string>
   </property>
  </action>
  <action name="actionReload_Files">
   <property name="text">
    <string>Reload Files</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_Copy</sender>
   <signal>triggered()</signal>
   <receiver>bCopy</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_HFlip</sender>
   <signal>triggered()</signal>
   <receiver>bHFlip</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_GPen</sender>
   <signal>triggered()</signal>
   <receiver>bGPEN</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Resize</sender>
   <signal>triggered()</signal>
   <receiver>bResize</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
